### PR TITLE
[Term Entry] Java Calendar isSet()

### PR DESCRIPTION
**Pull request by KingSky00**
### Description

- Added the Calendar .isSet() term for Java
- Closing issue #3104

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [ ] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/java/concepts/calendar/terms/isSet/isSet
Comment (SSwiniarski) [inline content/java/concepts/calendar/terms/isSet/isSet:1]: You need to remove this file from the PR.

File: content/java/concepts/calendar/terms/isSet/isSet.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+  - 'Computer Science'
+Tags:
+  - 'Methods'
+  - 'Calendar'
+CatalogContent:
+  - 'learn-java'
+  - 'paths/computer-science'
+---
+
+The **`.isSet(int calndr_field)`** method in `Calendar` class is used to check whether the given calendar field has a `value` set or not. All the `Cases` for which the values have been set by the calculations of the internal field are triggered by a `get` method call.
```
Comment (SSwiniarski) [inline content/java/concepts/calendar/terms/isSet/isSet.md:14]:
```suggestion
The **`.isSet()`** method in `Calendar` class is used to check whether the given calendar field has a value set or not. All the cases for which the values have been set by the calculations of the internal field are triggered by a `get` method call.
```
The second sentence is not that clear. Can you rephrase it so it is more obvious what you're trying to say?

File: content/java/concepts/calendar/terms/isSet/isSet.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+
+## Syntax
+
+```pseudo
+public final boolean isSet(int calndr_field)
```
Comment (SSwiniarski) [inline content/java/concepts/calendar/terms/isSet/isSet.md:19]:
```suggestion
result = myCalendar.isSet(calendar_field)
```
We like an example of the function in use rather than the signature.

File: content/java/concepts/calendar/terms/isSet/isSet.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+```
+
```
Comment (SSwiniarski) [inline content/java/concepts/calendar/terms/isSet/isSet.md:21]: Need some description of the syntax. Especially  the possiblities for `calendar_field`.

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@KingSky00 I did a first review and left a few comments.

File: content/java/concepts/calendar/terms/isSet/isSet.md (hunk 1 above)
Comment (KingSky00) [reply content/java/concepts/calendar/terms/isSet/isSet.md]: Thank you for the feedback. I hope that it looks better with the new changes!

File: content/java/concepts/calendar/terms/isSet/isSet.md [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+CatalogContent:
+  - 'learn-java'
+  - 'paths/computer-science'
+---
+
+The **`.isSet()`** method in `Calendar` class is used to check whether the given calendar field has a value set or not. This method returns `true` if the given calendar field has a value set, `false` otherwise. It returns true when the given parameter has been set by internal field interpretations. `isSet()` method is a non-static method, it is accessible with the class object.
+
+
+
+## Syntax
```
Comment (SSwiniarski) [inline content/java/concepts/calendar/terms/isSet/isSet.md:18]:
```suggestion
The **`.isSet()`** method in `Calendar` class is used to check whether the given calendar field has a value set or not. This method returns `true` if the given calendar field has a value set, `false` otherwise. It returns true when the given parameter has been set by internal field interpretations. `isSet()` method is a non-static method, it is accessible with the class object.

## Syntax
```
Remove extra lines.

File: content/java/concepts/calendar/terms/isSet/isSet.md [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+
+```pseudo
+result = myCalendar.isSet(calendar_field)
+```
+
+**Parameters:** The method takes one parameter calndr_field that refers to the calendar field which is to be operated upon.
+**Return Value:** The method either returns True if the value of the calendar field is set else it returns False.
```
Comment (SSwiniarski) [inline content/java/concepts/calendar/terms/isSet/isSet.md:25]:
```suggestion
Parameters: The method takes one parameter `calendar_field` that refers to the calendar field which is to be operated upon.
Return Value: The method either returns `true` if the value of the calendar field is set, else it returns `false`.
```

File: content/java/concepts/calendar/terms/isSet/isSet.md [hunk 6]
```diff
@@ -0,0 +1,63 @@
...
+  - 'Computer Science'
+Tags:
+  - 'Methods'
+  - 'Calendar'
+CatalogContent:
+  - 'learn-java'
+  - 'paths/computer-science'
+---
+
+The **`.isSet()`** method in `Calendar` class is used to check whether the given calendar field has a value set or not. This method returns `true` if the given calendar field has a value set, `false` otherwise. It returns true when the given parameter has been set by internal field interpretations. `isSet()` method is a non-static method, it is accessible with the class object.
```
Comment (caupolicandiaz) [inline content/java/concepts/calendar/terms/isSet/isSet.md:14]: This definition is an exact match to other documentation, such as the link below. Whether this was intentional or not, this is plagiarism. There is no plagiarism permitted in docs. All entries must be original. Please revise the content to use your own words.
https://www.geeksforgeeks.org/calendar-isset-method-in-java-with-examples/#:~:text=The%20isSet(int%20calndr_field)%20method,by%20a%20get%20method%20call.

File: content/java/concepts/calendar/terms/isSet/isSet.md [hunk 7]
```diff
@@ -0,0 +1,63 @@
...
+
+## Syntax
+
+```pseudo
+result = myCalendar.isSet(calendar_field)
+```
+
+Parameters: The method takes one parameter calndr_field that refers to the `calendar_field` which is to be operated upon.
```
Comment (caupolicandiaz) [inline content/java/concepts/calendar/terms/isSet/isSet.md:22]:
```suggestion
Parameters: The method takes one parameter, `calendar_field`, that refers to the specific field to be operated upon.
```

File: content/java/concepts/calendar/terms/isSet/isSet.md [hunk 8]
```diff
@@ -0,0 +1,63 @@
...
+Return Value: The method either returns `true` if the value of the calendar field is set, else it returns `false`.
```
Comment (caupolicandiaz) [inline content/java/concepts/calendar/terms/isSet/isSet.md:23]:
```suggestion
Return Value: The method either returns `true` if the value of the field is set, else it returns `false`.
```

File: content/java/concepts/calendar/terms/isSet/isSet.md [hunk 9]
```diff
@@ -0,0 +1,63 @@
...
+
+> **Note:** Please note that the `Calendar` class has been replaced by the `LocalDate`, `LocalTime`, and `LocalDateTime` classes in the Java 8 and later Date and Time API. These classes provide a more modern and user-friendly API for working with dates and times in Java.
```
Comment (caupolicandiaz) [inline content/java/concepts/calendar/terms/isSet/isSet.md:25]:
```suggestion
> **Note:** The `Calendar` class has been replaced by the `LocalDate`, `LocalTime`, and `LocalDateTime` classes in the Java 8 and later Date and Time API. These classes provide a more modern and user-friendly API for working with dates and times in Java.
```

File: content/java/concepts/calendar/terms/isSet/isSet.md [hunk 10]
```diff
@@ -0,0 +1,63 @@
...
+
+## Example
+
+The following example shows the usage of `java.util.calendar.isSet()` method.
+
+```java
+import java.util.*;
```
Comment (caupolicandiaz) [inline content/java/concepts/calendar/terms/isSet/isSet.md:32]: examples should used 2 spaces not tabs and comments should begin with a capital
// Create a calendar

**Review by caupolicandiaz — CHANGES_REQUESTED** [review]
@KingSky00 I've added my initial comments. Please review carefully and update the entry as needed. Tag me when you've finished making changes. Thanks

**Review by caupolicandiaz — APPROVED** [review]
@KingSky00 @SSwiniarski Looks good to go
